vignettes/howto-dm-rows.Rmd

```diff
@@ -19,25 +19,25 @@ source("setup/setup.R")
 ## Introduction {#intro}
 
 This tutorial introduces the methods {dm} provides for modifying the data in the tables of a relational model.
-There are 6 methods:
+There are 5 methods:
 
 * [`dm_rows_insert()`](#insert) - adds new rows
 * [`dm_rows_update()`](#update) - changes values in rows
 * [`dm_rows_patch()`](#patch) - fills in missing values
 * [`dm_rows_upsert()`](#upsert) - adds new rows or changes values if pre-existing
 * [`dm_rows_delete()`](#delete) - deletes rows
-* [`dm_rows_truncate()`](#truncate) - removes all rows, leaving table structure intact
 
 ## The dm_rows_* process
 
 All six methods take the same arguments and using them follows the same process:
 
-1. Create a temporary *changeset dm* that defines the intended changes on the RDBMS
+1. Create a temporary *changeset dm* object that defines the intended changes on the RDBMS <!-- TODO The technical term "changeset" needs to be explained here -->
```

Comment:
is there an issue for this TODO?

Reply:
Done!
https://github.com/cynkra/dm/issues/1141

Reply:
Thank you!